Source/FullScreenPokemon.ts

```diff
@@ -10459,12 +10459,12 @@ module FullScreenPokemon {
         }
 
         /**
-         * Displays an error message to the user.
+         * Displays a message to the user.
          *
          * @param thing   The Thing that triggered the error.
          * @param message   The message to be displayed.
          */
-        errorMessage(thing: IThing, message: string): void {
+        displayMessage(thing: IThing, message: string): void {
```

Comment:
Now that this is a generic utility, there's even less reason to have it take in `thing`. Just use `this` instead of `thing.FSP`.